doc-Managing_Providers/topics/Adding_Amazon_EC2_Providers.adoc

```diff
@@ -1,3 +1,4 @@
+[[adding-amazong-ec2-providers]]
```

Comment:
Tiny nitpick - typo in xref link. I didn't want to fix it in case it breaks something though :)